actors/gugu-mbatha-raw.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Gugu Mbatha-Raw",
+    "birthname": "Gugulethu Sophia Mbatha",
+    "birthdate": "1983-04-21",
+    "birthplace": "Oxford, Oxfordshire, England, UK"
+  }
+  
```

Comment:
Remove the extra line and bring the closing curly brace left 1 space

Reply:
Done! :)